Flank/src/main/java/com/walmart/otto/reporter/PriceReporter.java

```diff
@@ -0,0 +1,49 @@
+package com.walmart.otto.reporter;
+
+
+import java.math.BigDecimal;
+import java.text.ParseException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class PriceReporter {
+
+    private static HashMap<String, Double> mapOfPrices = new HashMap<String, Double>();
+    private static HashMap<String, BigDecimal> estimates = new HashMap<String, BigDecimal>();
+
+
+    public static HashMap<String, BigDecimal>  getTotalPrice(List<Integer> times)  {
+        mapOfPrices.put("virtual", (double)1);
```

Comment:
I did not know where to create the Hashmap. Ideally this should be created somewhere in a Util file . Any suggestion here?